apps/web/pages/availability/index.tsx

```diff
@@ -186,6 +188,16 @@ export default function AvailabilityPage() {
     },
     [searchParams]
   );
+
+  const isOrgAdminOrOwner =
+    data && (data.user.role === MembershipRole.OWNER || data.user.role === MembershipRole.ADMIN);
+  const isOrgAndPrivate = data?.isOrganization && data.isPrivate;
+  const toggleGroupOptions = [{ value: "mine", label: t("my_availability") }];
+
+  if (!isOrgAndPrivate || isOrgAdminOrOwner) {
+    toggleGroupOptions.push({ value: "team", label: t("team_availability") });
+  }
+
```

Comment:
NIT: should we make this a helper function? in general it would be nice to have something like role=owner, role=member, role=admin, etc.

Reply:
follow up PR if needed